src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -60,11 +61,11 @@ public AsyncQueue<Diagnostic> DiagnosticQueue
 
         /// <summary>
         /// Initializes the compilation for the analyzer driver.
-        /// It also computes and initializes <see cref="compilationAnalysisScope"/> and <see cref="_symbolActionsByKind"/>.
+        /// It also computes and initializes <see cref="analyzerActions"/> and <see cref="_symbolActionsByKind"/>.
         /// Finally, it initializes and starts the <see cref="_primaryTask"/> for the driver.
         /// </summary>
         /// <remarks>
-        /// NOTE: This method must only be invoked from <see cref="AnalyzerDriver.Create(Compilation, ImmutableArray{DiagnosticAnalyzer}, AnalyzerOptions, out Compilation, CancellationToken)"/>.
+        /// NOTE: This method must only be invoked from <see cref="AnalyzerDriver.Create(Compilation, ImmutableArray{DiagnosticAnalyzer}, AnalyzerOptions, AnalyzerManager, out Compilation, CancellationToken)"/>.
         /// </remarks>
         private void Initialize(Compilation comp, CancellationToken cancellationToken)
```

Comment:
if eventually we will share AnalyzerDriver, I think we need something that control concurrency. I mean any concurrency such as running two different analyzer concurrently.

like I said in previous comment, we already tried that, and that caused too much GC to happen since allocation rate spike up. IDE side, we should not run anything concurrently except user initiated actions such as fix all or ctrl+.

as you might see from my previous email, just having higher priority queue with no concurrency made us to miss 2-core typing goal. having concurrency in IDE with higher priority queue might make us to even miss 4-core typing goal.